nixos/modules/config/networking.nix

```diff
@@ -59,6 +64,14 @@ in
       '';
     };
 
+    networking.resolvconfOptions = lib.mkOption {
+      type = types.lines;
```

Comment:
You're right, seems like it would simplify the processing of the option value, too.